app/build.gradle

```diff
@@ -61,12 +61,41 @@ android {
         }
     }
 
+    lintOptions {
+        lintConfig file("lint.xml")
+    }
+
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
 }
 
+android.applicationVariants.all { variant ->
+
+// -------------------------------------------------------------------------------------------------
+// Sentry: Read token from locale file if it exists (Only release builds)
+// -------------------------------------------------------------------------------------------------
+    print("Sentry token:  "+ variant.name)
+    try {
+        def token = new File("${rootDir}/.sentry_token").text.trim()
+        buildConfigField 'String', 'SENTRY_TOKEN', '"' + token + '"'
+        println "(Added from .sentry_token file)"
+    } catch (FileNotFoundException ignored) {
+        buildConfigField 'String', 'SENTRY_TOKEN', 'null'
+        println(" :( ")
+    }
+
+// -------------------------------------------------------------------------------------------------
+// Activating crash reports with command line parameter.
+// -------------------------------------------------------------------------------------------------
+    if (project.hasProperty("crashReport") && project.property("crashReport") == "true") {
+        buildConfigField 'boolean', 'CRASH_REPORT_ON', 'true'
```

Comment:
nit: maybe `CRASH_REPORT_ON` -> `CRASH_REPORTING_ENABLED`? Maybe the same for the property? `crashReporting`?